src/components/Tool/Appeal/AppealDetails/EditAppealHeaderInfoModal/EditAppealHeaderInfoModal.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { ReactElement } from 'react';
+import {
+  DialogActions,
+  DialogContent,
+  FormHelperText,
+  TextField,
+} from '@mui/material';
+import { Box } from '@mui/system';
+import { Formik } from 'formik';
+import { useSnackbar } from 'notistack';
+import { useTranslation } from 'react-i18next';
+import * as yup from 'yup';
+import { AppealFieldsFragment } from 'pages/accountLists/[accountListId]/tools/GetAppeals.generated';
+import { FieldWrapper } from 'src/components/Shared/Forms/FieldWrapper';
+import {
+  CancelButton,
+  SubmitButton,
+} from 'src/components/common/Modal/ActionButtons/ActionButtons';
+import Modal from 'src/components/common/Modal/Modal';
+import { useAccountListId } from 'src/hooks/useAccountListId';
+import {
+  AppealDocument,
+  AppealQuery,
+} from '../AppealsMainPanel/appealInfo.generated';
+import { useUpdateAppealMutation } from './EditAppeal.generated';
+
+interface EditAppealHeaderInfoModalProps {
+  handleClose: () => void;
+  appealInfo: AppealFieldsFragment;
+}
+
+export type EditAppealFormikSchema = {
+  name: string;
+  amount: number;
+};
+
+const EditAppealSchema: yup.SchemaOf<EditAppealFormikSchema> = yup.object({
+  name: yup.string().required('Please enter a name'),
+  amount: yup.number().required('Please enter a goal'),
```

Comment:
Maybe we could force the goal amount to be a positive number? Not necessary though.